app/models/edition.rb

```diff
@@ -88,6 +89,7 @@ def change_note_html
   def draft_copy
     dup.tap do |e|
       e.change_note = nil
+      e.change_reason_for_change = nil
```

Comment:
They're no longer used. You can delete `Guide#ensure_draft_exists` and `Edition#draft_copy` along with tests for both.